docs/installation/windows.md

```diff
@@ -202,7 +203,6 @@ If this fails that is because mingw-w64-x86_64-qt5 was not installed. If you did
 * Boost_FILESYSTEM_LIBRARY_DEBUG/RELEASE C:/msys64/mingw64/bin/libboost_filesystem-mt.dll

```

Comment:
Is checking the filesystem and system variables still needed?

Reply:
the boost unit test library still needs them,  But I suppose we could remove the reference since they should get included in the test library and are not called out directly anymore.